### PR TITLE
Improve README to better explain binary

### LINKED ISSUE #397: Improve README documentation surrounding native binary
Remove "running with Java" from the extension description, and explain that a binary version or the Java version can be used.

Explain that the extension downloads the appropriate binary version of LemMinX if the user doesn't have Java.

### PR DESCRIPTION
**Pull request by datho7561**
Closes #397

Signed-off-by: David Thompson <davthomp@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ or you can read this documentation inside vscode with the command `Open XML Docu
 For running the binary version:
   * Windows, macOS, or Linux, on a x86_64 CPU
     * We don't make a binary specific for Apple ARM (Apple Silicon), but the x86_64 binary seems to work through the Rosetta 2 translation layer.
+  * Java is not required for this version
+  * The binary is automatically downloaded by vscode-xml if it is needed, with no additional action required on the part or the user.
```
Comment (datho7561) [inline README.md:48]: oops

**Review by rgrunber — COMMENTED** [review]
Overall, I'm fine with these changes. I just have a few questions on some other things I noticed.

- Is https://github.com/redhat-developer/vscode-xml/blob/master/docs/Preferences.md#java-home going to be updated separately or is it worth including it in here ?
- The first supported setting mentioned says "Set the Java path required to run the XML Language Server." It needs to be updated but also, I think, given that this is the first setting someone may see when viewing the README, it should probably be reworded to emphasize that this isn't required any longer.

**Comment by datho7561** [comment]
Good catch, one second. I'll amend

**Review by rgrunber — APPROVED** [review]
Looks good to me. Feel free to merge this.

**Pull request merged**